.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+ - "stable"
+services:
+ - mongodb
+ script: npm test
```

Comment:
Always put a new line at the end of every file, otherwise you will see this red arrow. The new line helps git.

Reply:
okay sir,that done in the new pull request sir.